sanic/request.py

```diff
@@ -61,7 +62,10 @@ def json(self):
             try:
                 self.parsed_json = json_loads(self.body)
             except Exception:
-                pass
+                return HTTPResponse(json_dumps(self.body),
```

Comment:
there is a really bad idea. I don't expect to get the response in case to access the `request.json`